primitives/teerex/src/lib.rs

```diff
@@ -22,6 +22,8 @@ use scale_info::TypeInfo;
 use sp_core::H256;
 use sp_std::prelude::*;
 
+pub const TEEREX: &str = "TEEREX";
```

Comment:
I don't actually know if there is a specific reason, but in other codebases this seems to be lowercase.